shared/transforms/combineremediations.py

```diff
@@ -370,9 +370,11 @@ def main():
                                              'platform', 'reboot', 'strategy']:
                                 config[key.strip()] = value.strip()
                             else:
-                                mod_file += line
+                                if not '# THIS FILE IS GENERATED' in line:
```

Comment:
Sorry for nitpicking again but since we already know line starts with `#` we might as well use.

```
if not line.startswith("# THIS FILE IS GENERATED")
```

The current version will match substrings in any part of the string. I don't think it's really an issue in practice, it's more of a code readability issue.

It would also help if the string literal was in a constant variable for reuse.